Tests/OpenCombineTests/PublisherTests/DecodeTests.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  DecodeTests.swift
+//  
+//
+//  Created by Joseph Spadafora on 6/21/19.
+//
+
+import XCTest
+
+#if OPENCOMBINE_COMPATIBILITY_TEST
+import Combine
+#else
+import OpenCombine
+#endif
+
+@available(macOS 10.15, *)
+final class DecodeTests: XCTestCase {
+    static let allTests = [
+        ("testDecodeWorks", testDecodeWorks),
+        ("testDownstraemReceivesFailure", testDownstreamReceivesFailure)
+    ]
+
+    private let jsonEncoder = JSONEncoder()
+    private let jsonDecoder = JSONDecoder()
+
+    let testValue = TestDecodable()
+
+    func testDecodeWorks() {
+        let data = try! jsonEncoder.encode(testValue)
```

Comment:
I know that this won't crash, but nothing on the type level says so. Crashing the test suite is not good, because it prevents further tests from execution. Let's mark the test method `throws` and remove this exclamation mark.